### DISCUSSION TITLE
0.20.9 - Water Hazard (hotfix)

### DISCUSSION THREAD

**Discussion by derrod** [legendary-gl/legendary#333] [General]
This is a hotfix release that fixes logging in as well as importing an existing login from EGL on Linux.

# Fixes and Improvements
- Fixed importing existing authentication from Epic Games Launcher on Linux #330 
- Fixed logging in via SID due to Epic API changes #329

<hr /><em>This discussion was created from the release <a href='https://github.com/derrod/legendary/releases/tag/0.20.9'>0.20.9 - Water Hazard (hotfix)</a>.</em>

**Comment by hisenburg411** [comment]
Opens up and closes immediately does not work

**Comment by jepa1311** [comment]
the SID still doesnt work

**Reply by CommandMC** [reply]
The issues relating to SIDs have been fixed, so this looks like an individual issue.
Could you head over to our discord server to discuss this further?
